SUTrivBot/Models/Question.cs

```diff
@@ -100,8 +109,8 @@ public AnswerResponse VerifyAnswer(string answer)
                         answer.Contains(item, StringComparison.InvariantCultureIgnoreCase)))
                         response.AnswerStatus = AnswerStatus.PartiallyCorrect;
                 }
-                else if (Answers.Any(item =>
-                    answer.Contains(item, StringComparison.InvariantCultureIgnoreCase)))
+                else if (Answers.Count(item =>
+                    answer.Contains(item, StringComparison.InvariantCultureIgnoreCase)) >= AnswersRequired)
                     response.AnswerStatus = AnswerStatus.PartiallyCorrect;
 

```

Comment:
With the change in conditionals for being correct two things should be changed here. 
the condition where the answer contains at least as many answers as is required should set the answer status to NormalCorrect
Then there should be another else if to determine if they got ANY answers correct and be given a partially Correct